hogql_parser/setup.py

```diff
@@ -32,7 +32,7 @@
 
 setup(
     name="hogql_parser",
-    version="1.0.4",
+    version="1.0.5",
```

Comment:
I've already used this version up in https://github.com/PostHog/posthog/pull/21203 😄 
The current workflow does not lend itself to multiple PRs around hogql-parser being worked on concurrently